src/Form/tests/FormAutosuggest.test.jsx

```diff
@@ -166,31 +166,49 @@ describe('controlled behavior', () => {
 
     fireEvent.click(input);
     fireEvent.change(input, { target: { value: 'Option 1' } });
-    
+
     const list = container.querySelectorAll('li');
     expect(list.length).toBe(1);
   });
 
-  //     it('toggles options list', () => {
-  //      this is toggling when the dropdown button is clicked
-  //       const dropdownContainer = '.pgn__form-autosuggest__dropdown';
+  it('toggles options list', () => {
+    //  this is toggling when the dropdown button is clicked
+    // const dropdownContainer = '.pgn__form-autosuggest__dropdown';
 
-  //       expect(container.find(dropdownContainer).find('button').length).toEqual(3);
+    // expect(container.find(dropdownContainer).find('button').length).toEqual(3);
 
-  //       container.find('button.pgn__form-autosuggest__icon-button').simulate('click');
-  //       expect(container.find(dropdownContainer).find('button').length).toEqual(0);
+    // container.find('button.pgn__form-autosuggest__icon-button').simulate('click');
+    // expect(container.find(dropdownContainer).find('button').length).toEqual(0);
 
-  //       container.find('button.pgn__form-autosuggest__icon-button').simulate('click');
-  //       expect(container.find(dropdownContainer).find('button').length).toEqual(3);
-  //     });
+    // container.find('button.pgn__form-autosuggest__icon-button').simulate('click');
+    // expect(container.find(dropdownContainer).find('button').length).toEqual(3);
+
+    const { container } = render(<FormAutosuggestTestComponent />);
+    const dropdownBtn = container.querySelector('button.pgn__form-autosuggest__icon-button');
+
+    fireEvent.click(dropdownBtn);
+
+    const list = container.querySelectorAll('li');
+    expect(list.length).toBe(3);
+
+    fireEvent.click(dropdownBtn);
+    // Re-query the list of elements again after the second click
+    const updatedList = container.querySelectorAll('li');
+
+    expect(updatedList.length).toBe(0);
+
+    // is this last bit necessary?
+    fireEvent.click(dropdownBtn);
+    expect(list.length).toBe(3);
```

Comment:
I believe the goal here is to test that it still opens the dropdown after it's been closed using the button before. I'm not 100% sure that's properly being tested here, what happens if you put the expect before the final click?